lighthouse-core/gather/connections/connection.js

```diff
@@ -111,7 +98,7 @@ class Connection {
       // just log these occurrences.
       const error = object.error && object.error.message;
       log.formatProtocol(`disowned method <= browser ${error ? 'ERR' : 'OK'}`,
-          {method: object.method, params: error || object.result}, 'verbose');
+          {method: 'UNKNOWN', params: error || object.result}, 'verbose');
```

Comment:
do we not get back a method from disowned messages? seems like `|| 'UNKOWN'` would work if we do :)

Reply:
> do we not get back a method from disowned messages?

we can check on the Chrome side if it's possible, but I did a bunch of runs watching the properties of all messages sent back over the protocol and never got a `method` on a non-event response